op_crates/webgpu/lib.rs

```diff
@@ -113,73 +112,75 @@ pub fn get_declaration() -> PathBuf {
   PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("lib.deno_webgpu.d.ts")
 }
 
-fn deserialize_features(features: &wgpu_types::Features) -> Vec<&str> {
-  let mut return_features: Vec<&str> = vec![];
+fn deserialize_features(features: &wgpu_types::Features) -> Vec<String> {
+  let mut return_features: Vec<String> = vec![];
 
   if features.contains(wgpu_types::Features::DEPTH_CLAMPING) {
-    return_features.push("depth-clamping");
+    return_features.push("depth-clamping".to_string());
```

Comment:
Again, a bunch of unnecessary allocations. Can serde_v8 not handle `&str`.

Reply:
I converted this to use `&'static str` (which can be safely returned then serialized), but in practice this should have no performance impact given that this setup method should be called very infrequently.